### PR TITLE
Update documentation with migration changes

### LINKED ISSUE #23: Website guide is out of date
## Description

The website currently links to evaeras wally posting, it also says evaeras name for the wally template. 

## Expected behavior

It links to the matter-ecs and uses matter-ecs in the template.

### PR DESCRIPTION
**Pull request by ATK422**
Changed everything from evaera/matter to matter-ecs/matter. Reworked the installation guide. Added Migration steps.

## Proposed changes

I went through and reworked some of the documentation to reflect changes to the org.

## Related issues
Resolves #23

### CODE REVIEW

**Review by jackTabsCode — APPROVED** [review]

**Comment by ATK422** [comment]
All the changes look good

**Comment by LastTalon** [comment]
> All the changes look good

It looks like you just resolved the issues without merging any suggested changes. Could you try to unresolve them to see if you can add them?

**Comment by ATK422** [comment]
Should be fixed now, sorry about that.

File: docs/Installation.md [hunk 1]
```diff
@@ -4,34 +4,34 @@ sidebar_position: 2
...
-realm = "shared"
+4. If you don't have a `wally.toml` file, run `wally init`.
 
+5. Add matter under `[dependencies]` inside `wally.toml`. You can find the latest version from [this page](https://wally.run/package/matter-ecs/matter) or by running `wally search matter-ecs` and finding the one labeled `matter-ecs/matter@x.x.x`.
+
+```toml title="wally.toml"
 [dependencies]
-matter = "evaera/matter@X.X.X" # Don't copy this. This won't work.
-                               # Copy real string from page linked above.
+matter = "matter-ecs/matter@x.x.x" # Replace x.x.x with the latest version
```
Comment (jackTabsCode) [inline docs/Installation.md:28]: Looks like you forgot this one

Comment (LastTalon) [reply docs/Installation.md:28]: I guess I forgot that one. Same deal.

```suggestion
matter = "matter-ecs/matter@0.7.0"
```

**Comment by ATK422** [comment]
Alright, committed it.

**Review by Ukendio — APPROVED** [review]

**Review by LastTalon — APPROVED** [review]
Seems like this should be fine to me.

**Pull request merged**